BackofficeBundle/Controller/SiteController.php

```diff
@@ -41,6 +43,9 @@ public function formAction(Request $request, $siteId)
         $form->handleRequest($request);
         $message =  $this->get('translator')->trans('open_orchestra_backoffice.form.website.success');
         if ($this->handleForm($form, $message)) {
+            if($site->getAliases()->toArray() !== $aliases) {
+                $this->dispatchEvent(SiteAliasEvents::SITEALIAS_UPDATE, new SiteAliasEvent($site));
```

Comment:
Right. but If I use SITE_EVENT, I just receive the new site whithout knowing if aliases were changed or not. In this case, I've got two solutions, change event to pass old site too or refresh route even if aliases didn't change. That's why, I made this choice. @nicolasThal